subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/Gradle.kt

```diff
@@ -26,10 +31,33 @@ internal object CILoggerRegistry {
 
 private fun provideLogger(project: Project, loggerName: String): CILogger {
     return if (project.buildEnvironment is BuildEnvironment.CI && !project.buildEnvironment.inGradleTestKit) {
+        val endpoints: List<String> =
+            requireNotNull(
+                project.properties["avito.elastic.endpoints"]
+                    ?.toString()
+                    ?.split("|")
+            ) {
+                "avito.elastic.endpoints has not been provided"
```

Comment:
What if I don't want to use it? (don't have or whatever)

Reply:
`avito.elastic.enabled` added